solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1045,15 +1044,25 @@ impl parachains_scheduler::common::AssignmentProvider<BlockNumberFor<Runtime>>
     }
 }
 
+#[cfg(feature = "fast-runtime")]
+pub const TIMESLICE_PERIOD: u32 = 20;
+#[cfg(not(feature = "fast-runtime"))]
+pub const TIMESLICE_PERIOD: u32 = 80;
+
 parameter_types! {
     pub const OnDemandTrafficDefaultValue: FixedU128 = FixedU128::from_u32(1);
+    // Keep 2 timeslices worth of revenue information.
+    pub const MaxHistoricalRevenue: BlockNumber = 2 * TIMESLICE_PERIOD;
+    pub const OnDemandPalletId: PalletId = PalletId(*b"py/ondmd");
 }
 
 impl parachains_assigner_on_demand::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
     type Currency = Balances;
     type TrafficDefaultValue = OnDemandTrafficDefaultValue;
     type WeightInfo = parachains_assigner_on_demand::TestWeightInfo;
+    type MaxHistoricalRevenue = MaxHistoricalRevenue;
+    type PalletId = OnDemandPalletId;
```

Comment:
what is MaxHistoricalRevenue? does it affect us (that we dont have a notion of timeslices?). can we put it to ()?

Reply:
I don't know, I copied it from polkadot. Will check

Reply:
Changed it to 2. It's used as a bound for a BoundedVec and it cannot be 0 because the pallet assumes that it's at least 1 so we will see warnings and it may break things. I have set it to 2 in case I missed something.

Reply:
_fixed_